middleware/multer.js

```diff
@@ -0,0 +1,17 @@
+const multer = require('multer');
```

Comment:
Same remark as above? Is multer being used in the application

Reply:
Yes sir, here multer is used for temporary disk storage with a unique name before finally uploading the file to the Cloudinary platform. This function is then further imported and used in the upload route handling file.

Reply:
Okay